### DISCUSSION TITLE
ubuntu下没有自启

### DISCUSSION THREAD

**Discussion by kilvn** [AlistGo/alist#2512] [General]
Ubuntu 22.04.1 LTS

每次重启后都得手动执行  启动命令
systemctl start alist.service

看了 service 启动文件，没毛病啊
execStart 的命令没什么问题，但是就是每次重启服务器都得手动去启动服务，没找到问题在哪。

**Comment by yage-zhu** [comment]
systemctl enable alist.service

**Reply by kilvn** [reply]
这个执行好几次了，没有用

**Comment by kilvn** [comment]
已经找到原因了，改成了docker mysql，由于开机后mysql还没有那么快启动起来，所以导致系统服务启动失败了，解决方法就是给系统服务增加延迟多少秒后启动和失败重试机制。

我取消了系统服务，用supervisor统一管理，做守护了，问题解决。
